src/cookbooks/bigquery-airflow.md

```diff
@@ -27,46 +26,44 @@ WHERE
 
 `@submission_date` is a parameter that will be filled in by Airflow.
 
-Commit changes in `/sql`. 
-When a commit is made to master in `bigquery-etl`, the Docker image is pushed and available to Airflow.
-
-## In telemetry-airflow
-
-The next step is to create a DAG or add a task to an existing DAG that will run the query.  
-In [telemetry-airflow](https://github.com/mozilla/telemetry-airflow), BigQuery related functions are found in `/dags/utils/gcp.py`.
-The function we are interested in is [`bigquery_etl_query`](https://github.com/mozilla/telemetry-airflow/blob/c103f3eee4ddc653316325d0ee0deab0bb35ee57/dags/utils/gcp.py#L390).
-
-For our `client_ids` example, we could create a new DAG, `/dags/client_ids.py`:
-```python
-from airflow import models
-from utils.gcp import bigquery_etl_query
-
-default_args = {
-    ...
-}
-
-dag_name = 'client_ids'
+To schedule the create query, create a `/sql/example/client_ids/metadata.yaml` file with the following content:
```

Comment:
I guess this will be replaced soon with the doc describing how to create new datasets. But in the meantime it's still nice to have the docs relatively up-to-date